src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -148,6 +148,12 @@
   @#   A Channel object through which to make calls.
   @# @@param chan_creds [Grpc::ChannelCredentials]
   @#   A ChannelCredentials for the setting up the RPC client.
+  @# @@param updater_proc [Proc]
+  @#   A function that transforms the metadata for requests, e.g., to give
```

Comment:
I think the terminology of updater_proc is from `googleauth`: https://github.com/google/google-auth-library-ruby/blob/37ed189b2e5165243918fcde127ebe323e9a06b5/lib/googleauth/signet.rb#L58